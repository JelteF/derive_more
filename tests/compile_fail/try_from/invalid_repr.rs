#[derive(derive_more::TryFrom)]
#[try_from(repr)]
#[repr(a + b)]
enum Enum {
    Variant
}

fn main() {}
