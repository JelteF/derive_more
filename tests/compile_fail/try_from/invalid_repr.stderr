error: expected `,`
 --> tests/compile_fail/try_from/invalid_repr.rs:2:10
  |
2 | #[repr(a + b)]
  |          ^

error: expected one of `(`, `,`, `::`, or `=`, found `+`
 --> tests/compile_fail/try_from/invalid_repr.rs:2:10
  |
2 | #[repr(a + b)]
  |          ^ expected one of `(`, `,`, `::`, or `=`
